tests/unit/lib/test_signal_schema.py

```diff
@@ -356,6 +357,24 @@ def test_slice_nested():
     assert list(sliced.values.items()) == [("feature.aa", int)]
 
 
+def test_mutate_rename():
+    schema = SignalSchema({"name": str})
+    schema = schema.mutate({"new_name": Column("name")})
+    assert schema.values == {"new_name": str}
+
+
+def test_mutate_new_signal():
+    schema = SignalSchema({"name": str})
+    schema = schema.mutate({"age": Column("age", Float)})
```

Comment:
It should throw an exception (missing signal), shouldn't it?

Reply:
`SignalSchema.mutate()` was from beginning implemented to accept new signals so this should not throw exception, instead it will just add new signal.